addons/gamification/views/gamification_goal_definition_views.xml

```diff
@@ -34,11 +34,11 @@
                 <sheet>
                         <label for="name"/>
                         <h1>
-                            <field name="name" placeholder="e.g. Get started"/>
+                            <field name="name" placeholder="e.g. Get started" style="width:100%;"/>
```

Comment:
could yo use a bootstrap class having the same purpose ?

Reply:
updted